cli/crates/cli/tests/compilation_error.rs

```diff
@@ -37,12 +39,39 @@ fn compilation_error() {
 
     let response = client.gql::<Value>(DEFAULT_QUERY).send();
 
-    let errors: Option<Vec<String>> = dot_get_opt!(response, "errors");
+    let errors: Option<Vec<Value>> = dot_get_opt!(response, "errors");
```

Comment:
Are these not strings?

Reply:
No. https://spec.graphql.org/October2021/#example-072c4